provision/lib/Warewulf/Provision/Pxe.pm

```diff
@@ -100,6 +100,9 @@ setup()
                     &eprint("Could not locate Warewulf's internal $datadir/warewulf/ipxe/$f! Things might be broken!\n");
                 }
             }
+            if (-f "$tftpdir/warewulf/ipxe/$f") {
+                system("ln -sf $tftpdir/warewulf/ipxe/$f $tftpdir/warewulf/ipxe/$f.0");
```

Comment:
I'd prefer we not bake this `*.0` specific fix into Warewulf. This workaround seems like something that should be put into site location configuration management.

Reply:
Maybe just a "boot image suffix" setting in provision.conf which is used when copying the images?

Reply:
Even that seems like something that shouldn't be in Warewulf.

Reply:
Well, the boot images are managed/provided by WW. The problem with my machines is, that they explicitly request "image.0" when I configure "image". Isc-dhcp seems to just try to find the file without the suffix, but I haven't figured out yet how to do that with dnsmasq.

Reply:
From the dnsmasq main page for pxe-service: ```Note that the "layer" suffix (normally ".0") is supplied by PXE, and need not be added to the basename. Alternatively, the basename may be a filename, complete with suffix, in which case no layer suffix is added.```

Can you try using dhcp-boot instead of pxe-service?